compiler/modules/front.syntax/astString.bal

```diff
@@ -258,6 +258,12 @@ function typeDescToWords(Word[] w, TypeDesc td, boolean|BinaryTypeOp wrap = fals
         w.push(CLING, ">");
         return;
     }
+    else if td is ErrorTypeDesc {
+        w.push("map", CLING, "<", CLING);
```

Comment:
It's usually a record. Why aren't we calling typeDescToWords on the detail?

Reply:
I have made a mistake, this should be `error`.